colonist-samples/modular-android/modular-app/src/test/java/com/joom/colonist/modular/FeatureManagerTest.kt

```diff
@@ -16,13 +16,21 @@
 
 package com.joom.colonist.modular
 
-import com.joom.colonist.ColonistException
+import com.joom.colonist.modular.feature1.Feature1
+import com.joom.colonist.modular.feature2.Feature2
+import org.junit.Assert
 import org.junit.Test
 
 class FeatureManagerTest {
-  @Test(expected = ColonistException::class)
+  @Test
   fun testFeatureManagerIsNotProcessed() {
```

Comment:
I guess it should be `testFeatureManagerIsProcessed` in this case

Reply:
Sure